Tests/Tests/AppStartEventSpec.swift

```diff
@@ -0,0 +1,26 @@
+import Quick
+import Nimble
+@testable import RInAppMessaging
+import CoreFoundation
+
+class AppStartEventSpec: QuickSpec {
+    override func spec() {
+        describe("AppStartEvent") {
+            let appStartEvent = AppStartEvent()
+            context("AppStartEvent.analyticsParameters") {
+                it("will return dictionary with eventName and timestamp values") {
+                    if let eventName = appStartEvent.analyticsParameters["eventName"] as? String {
```

Comment:
please don't unwrap `eventName` and call `expect(eventName).to(equal("app_start"))` with an optional value. If the event name is not a String, the expectation should fail. Please apply the same for other files in this PR